### PR TITLE
Add build config to polymer.json

### PR DESCRIPTION
**Pull request by keanulee**

### CODE REVIEW

File: polymer.json [hunk 1]
```diff
@@ -19,5 +19,17 @@
...
     "manifest.json",
     "bower_components/webcomponentsjs/webcomponents-lite.min.js"
-  ]
+  ],
+  "builds": [{
+    "name": "bundled",
+    "bundle": true,
+    "js": {"minify": true},
+    "css": {"minify": true},
+    "html": {"minify": true}
```
Comment (frankiefu) [inline polymer.json]: "addServiceWorker": true

**Comment by frankiefu** [comment]
Hmm... tried to build it with `polymer build` and somehow the `manifest.json` is missing in the build.

**Comment by abdonrd** [comment]
@frankiefu make the following changes:

* `sourceGlobs` to `sources`
* `includeDependencies ` to `extraDependencies `

**Comment by abdonrd** [comment]
In fact, I think we should do the following:

```json
"sources": [
  "src/**/*",
  "images/**/*",
  "data/**/*",
  "bower.json"
],
"extraDependencies": [
  "app.yaml",
  "main.py",
  "manifest.json",
  "bower_components/webcomponentsjs/webcomponents-lite.min.js"
],
```

**Comment by abdonrd** [comment]
I have created https://github.com/Polymer/news/pull/138 😄

**Pull request merged**